From 190965b7ff6a90eee07ff37e489638c91f30a048 Mon Sep 17 00:00:00 2001
From: Eric Ernst <eric.ernst@intel.com>
Date: Fri, 5 Sep 2014 09:58:29 -0700
Subject: [PATCH 1/1] init_task: change slack from 50us to 1ms

Change the timer_slack from 50 usec to 1 msec
default slack

Signed-off-by: Eric Ernst <eric.ernst@intel.com>
---
 include/linux/init_task.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/init_task.h b/include/linux/init_task.h
index 6df7f9fe0d01..1ec52cd839b3 100644
--- a/include/linux/init_task.h
+++ b/include/linux/init_task.h
@@ -216,7 +216,7 @@ extern struct task_group root_task_group;
 	.journal_info	= NULL,						\
 	.cpu_timers	= INIT_CPU_TIMERS(tsk.cpu_timers),		\
 	.pi_lock	= __RAW_SPIN_LOCK_UNLOCKED(tsk.pi_lock),	\
-	.timer_slack_ns = 50000, /* 50 usec default slack */		\
+	.timer_slack_ns = 1000000, /* 1 msec default slack */		\
 	.pids = {							\
 		[PIDTYPE_PID]  = INIT_PID_LINK(PIDTYPE_PID),		\
 		[PIDTYPE_PGID] = INIT_PID_LINK(PIDTYPE_PGID),		\
-- 
1.9.1

